src/components/Toast/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { Toaster } from "react-hot-toast";
+import { CSSProperties } from "react";
+
+const Toast = () => {
+  const BasicToastStyles: CSSProperties = {
+    width: "180px",
+    padding: "8px 16px",
+    height: "auto",
+    borderRadius: "86px",
+    fontFamily: "Pretendard",
+    fontSize: "0.875rem",
+    fontWeight: 400,
+    lineHeight: "18.2px",
+    letterSpacing: "-0.02em",
+    textAlign: "center",
+  };
+
+  return (
+    <Toaster
+      position="top-center"
+      toastOptions={{
+        duration: 1500,
+        success: {
+          icon: null,
+          style: {
+            ...BasicToastStyles,
+            background: "#27272C",
+            color: "#F7F9FE",
+          },
+        },
+        error: {
+          icon: null,
+          style: {
+            ...BasicToastStyles,
+            background: "red",
```

Comment:
```suggestion
            background: "#FF0000",
```
다른 곳들도 다 hex code 사용해서 여기도 통일하면 좋을 것 같습니다~!